carts/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http     import JsonResponse
+from django.views    import View
+
+from products.models import Product
+from carts.models    import Cart
+from core.utils      import LoginConfirm
```

Comment:
로그인 데코레이터 함수명을 스네이크 케이스로 바꿔주세요.
```suggestion
from core.utils      import login_confirm
```